vault/resource_database_secret_backend_connection_test.go

```diff
@@ -956,20 +967,23 @@ func testAccDatabaseSecretBackendConnectionConfig_mssql(name, path, connURL stri
   }`
 	}
 
-	return fmt.Sprintf(`
+	result := fmt.Sprintf(`
 resource "vault_mount" "db" {
   path = "%s"
   type = "database"
 }
 
 resource "vault_database_secret_backend_connection" "test" {
   backend = vault_mount.db.path
+  plugin_name = "%s"
```

Comment:
Since we are explicitly setting `plugin_name` for MSSQL configs, would it be worth testing with a custom plugin name other than the default ones we are passing in wherever `testAccDatabaseSecretBackendConnectionConfig_mssql` is called? 

From an initial glance it looks like adding this field explicitly in MSSQL doesn't have a benefit since it always uses the default, but I could be wrong

Reply:
Resolved with https://github.com/hashicorp/terraform-provider-vault/pull/1320#discussion_r794779496